ci/script.sh

```diff
@@ -41,6 +41,7 @@ case $TARGET in
 
   lint)
     cargo fmt -- --check
+    taplo fmt --check
```

Comment:
Adding this as part of the `lint` step since that's what it is. I am planning on opening a follow up PR splitting these lint-related jobs from the `Tests` job into a separate `Lint` job.